data/talkactions/scripts/create_item.lua

```diff
@@ -8,10 +8,17 @@ function onSay(player, words, param)
 	end
 
 	local split = param:split(",")
+	if not split[1] then
```

Comment:
You should use `if #split < 1` instead, it is semantically better to check if the array has the correct amount of values rather than check an index.